Category-Colors/category-colors
# Update meta title, description & favicon

# Description
Update the placeholder meta text & favicon

# QA
Go to `http://localhost:3000/` and you should see the new title in the browser tab along with the favicon. 

![Screen Shot 2023-04-23 at 10 06 02 PM](https://user-images.githubusercontent.com/2035194/233883190-a03e63d7-0c5d-4a2f-8662-b71f909186f6.png)
